cypress/fixtures/invalid.json

```diff
@@ -0,0 +1,33 @@
+{
+    "firstName": "Nghia",
```

Comment:
you can combine into one fixture file with valid data like this:
```json
{
    "email": {
         "valid": "a@gmail.com",
         "invalid": [
              "@.com",
              "abcd@"
         ]
    }
}
```

You can access like `data.email.valid` and `data.email.invalid[0]`
Or you can use `data.email.invalid.forEach()` to loop through array